<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidgetEndpoints">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>191</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>230</y>
      <width>281</width>
      <height>309</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSendHttp">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>560</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Enviar solicitud</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Crear clase</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditClassName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Nombre</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPropertyName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Nombre propiedad</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDataType">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Tipo de dato</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAddProperties">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetShowClass">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>256</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
    <property name="verticalScrollMode">
     <enum>QAbstractItemView::ScrollPerPixel</enum>
    </property>
    <property name="horizontalScrollMode">
     <enum>QAbstractItemView::ScrollPerPixel</enum>
    </property>
    <item>
     <property name="text">
      <string>ClassName{</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>}</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>0</y>
      <width>134</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Crear endpoints</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxHttpMethods">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>134</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>GET</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>POST</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PUT</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DELETE</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBoxClasses">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>60</y>
      <width>134</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButtonAddClass">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Guardar clase</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAddEndpoint">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>150</y>
      <width>134</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar endpoint</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditEndpointText">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>120</y>
      <width>134</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>GET</string>
    </property>
    <property name="placeholderText">
     <string>Endpoint</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>0</y>
      <width>134</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Direccion Url</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditUrl">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>30</y>
      <width>134</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>http://127.0.0.1:8000</string>
    </property>
    <property name="placeholderText">
     <string>Url</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxProperties">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>90</y>
      <width>134</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>60</y>
      <width>134</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Url final</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditFinalUrl">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>90</y>
      <width>211</width>
      <height>81</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxClassEndpoints">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>191</width>
      <height>24</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidgetProperties">
    <property name="geometry">
     <rect>
      <x>206</x>
      <y>230</y>
      <width>301</width>
      <height>271</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>143</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Propiedad</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Valor</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Type</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButtonSetBodyRequest">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>510</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar cuerpo</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>510</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Estado:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelBodyRequestState">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>510</y>
      <width>82</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#labelBodyRequestState::Disabled{
color: red;
}

#labelBodyRequestState::Enabled{
color: green;
}</string>
    </property>
    <property name="text">
     <string>Sin cuerpo</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>190</y>
      <width>581</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonGET">
       <property name="text">
        <string>GET</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroupHttpMethods</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonPOST">
       <property name="text">
        <string>POST</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroupHttpMethods</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonPUT">
       <property name="text">
        <string>PUT</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroupHttpMethods</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonDELETE">
       <property name="text">
        <string>DELETE</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroupHttpMethods</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>lineEditPropertyName</sender>
   <signal>returnPressed()</signal>
   <receiver>lineEditDataType</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>66</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditDataType</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButtonAddProperties</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>66</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditClassName</sender>
   <signal>returnPressed()</signal>
   <receiver>lineEditPropertyName</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>66</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroupHttpMethods"/>
 </buttongroups>
</ui>
